Jenkinsfile

```diff
@@ -77,69 +21,57 @@ pipeline {
             steps {
-                installRpms()
+                script {
+                    if ( AUTHOR_NAME == 'null' )  {
+                        AUTHOR_NAME = sh(script: "git show -s --pretty=%ae | awk -F'@' '{print \$1}'", , returnStdout: true).trim()
+                    }  
+                }
                 sh '''
-                    if [ ! -d $HOME/google-cloud-sdk/bin ]; then
-                        rm -rf $HOME/google-cloud-sdk
-                        curl https://sdk.cloud.google.com | bash
-                    fi
-
-                    source $HOME/google-cloud-sdk/path.bash.inc
-                    gcloud components update kubectl
-                    gcloud version
-
-                    curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.14.0-linux-amd64.tar.gz \
-                        | sudo tar -C /usr/local/bin --strip-components 1 -zvxpf -
-                    curl -s -L https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz \
-                        | sudo tar -C /usr/local/bin --strip-components 1 --wildcards -zxvpf - '*/oc'
+                    curl -s -L https://github.com/mitchellh/golicense/releases/latest/download/golicense_0.2.0_linux_x86_64.tar.gz \
+                        | sudo tar -C /usr/local/bin --wildcards -zxvpf -
+                    curl -s -L https://github.com/src-d/go-license-detector/releases/latest/download/license-detector.linux_amd64.gz \
+                        | gunzip > license-detector
+                    sudo mv license-detector /usr/local/bin/license-detector
+                    sudo chmod +x /usr/local/bin/license-detector 
                 '''
             }
         }
-        stage('Build docker image') {
+        stage('GoLicenseDetector test') {
             when {
                 expression {
                     !skipBranchBulds
                 }
             }
             steps {
-                 withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
-                    sh '''
-                        DOCKER_TAG=perconalab/percona-xtradb-cluster-operator:master-broker-$VERSION
-                        docker_tag_file='./results/docker/TAG'
-                        mkdir -p $(dirname ${docker_tag_file})
-                        echo ${DOCKER_TAG} > "${docker_tag_file}"
-
-                        sg docker -c "
-                            docker login -u '${USER}' -p '${PASS}'
-                            export IMAGE=\$DOCKER_TAG
-                            ./e2e-tests/build
-                            docker logout
-                        "
-                    '''
-                }
-                stash includes: 'results/docker/TAG', name: 'IMAGE'
-                archiveArtifacts 'results/docker/TAG'
+               sh """
+                   IS_GPL=\$(license-detector ${WORKSPACE} | grep GPL) || true
```

Comment:
lets store allowed the output of checked here and `diff` it here

Reply:
done